src/commands/cmd_json.cc

```diff
@@ -52,7 +55,7 @@ class CommandJsonGet : public Commander {
   }
 };
 
-REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandJsonSet>("json.set", -3, "write", 1, 1, 1),
+REDIS_REGISTER_COMMANDS(MakeCmdAttr<CommandJsonSet>("json.set", 3, "write", 1, 1, 1),
```

Comment:
We should use `-3` after set with `NX` support.

Now I think we only support 3 arguments.

Reply:
Oops, seems this should be 4 (or -4 if implement NX and other options)